htlcswitch/link.go

```diff
@@ -533,7 +533,7 @@ func (l *channelLink) Start() error {
 		}()
 	}
 
-	l.updateFeeTimer = time.NewTimer(l.randomFeeUpdateTimeout())
+	l.updateFeeTimer = time.NewTimer(0)
```

Comment:
This basically means the timer will fire every 0 nanoseconds, so constantly. I think we need to instead reset the timer's delay after it's been fired.

Reply:
It should reset here: https://github.com/rkfg/lnd/blob/564c0572cfb2552ef19d704c9f587c95ffac4c32/htlcswitch/link.go#L1321

Timer only fires once, not constantly if I understand it correctly. After it expires it should be reset to work again which is exactly what happens before the fee update check.

Reply:
Ah, yes, you're correct. I think we should add a comment to just above the line you changed to make exactly that fact more obvious.

Reply:
Added a comment!